docs/fastdds/xml_configuration/common.rst

```diff
@@ -351,6 +352,61 @@ Please refer to :ref:`threadsettingsqos` for further information on ResourceLimi
      - ``int32_t``
      - -1
 
+.. _BuiltinTransportType:
+
+BuiltinTransports
+^^^^^^^^^^^^^^^^^
+
+It controls the builtin transports that will be used during the initialization of the DomainParticipant.
+Please refer to :ref:`rtps_layer_builtin_transports` for further information on builtin transports.
+
+This tag can be configured with the following attributes:
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Name
+    - Description
+    - Values
+    - Default
+  * - ``max_msg_size``
+    - Maximum message size that will be specified in
+      the transport layer. |br|
+      Valid values: from 1 to 2^32-1.
+    - ``uint32_t``
+    - 65500
+  * - ``sockets_size``
+    - Size of the send and receive socket buffers.
+      Valid values: from 0 to 2^32-1.
+    - ``uint32_t``
+    - OS default
+  * - ``non_blocking``
+    - Whether to use non-blocking send calls or not.
+      Valid values: true or false.
+    - ``bool``
+    - false
+  * - ``tcp_negotiation_timeout``
+    - Timeout duration for logical port negotiation.
+      Valid values: from 1 to 2^32-1. |br|
+      Only valid for ``LARGE_DATA`` mode.
+    - ``uint32_t``
+    - 0
+
+This type must contain the following tag:
```

Comment:
I would include this section right after the previous one.
the other would start:
_"The ``<builtinTransport>`` tag can be ..."_